addon/index.js

```diff
@@ -55,6 +55,11 @@ export default Mixin.create({
     const url = get(this, 'currentURL');
     const hashElement = url ? document.getElementById(url.split('#').pop()) : null
 
+    if (get(this, 'service.isFirstLoad')) {
+      get(this, 'service').unsetFirstLoad();
```

Comment:
So we are actually currently doing calling `scrollTo` on first load (however has no effect on first load for some reason on fast pages).  As detailed in the referenced issue, on first load, we should not try to scroll the page when we first enter the application.  On slow requests, this does show up from time to time.  

So using a flag `isFirstLoad`, we can prevent unnecessary work.

https://github.com/dollarshaveclub/ember-router-scroll/blob/master/addon/index.js#L75